components/textfield/index.css

```diff
@@ -231,35 +231,28 @@ governing permissions and limitations under the License.
   }
 
   .spectrum-Textfield.is-invalid & {
-    inline-size: var(--spectrum-icon-alert-medium-width);
-    block-size: var(--spectrum-icon-alert-medium-height);
+    inline-size: var(--spectrum-textfield-validation-icon-width-error);
+    block-size: var(--spectrum-textfield-validation-icon-height-error);
     inset-block-end: calc(
-      calc(var(--spectrum-textfield-validation-icon-height) / 2) -
-        calc(var(--spectrum-icon-alert-medium-height) / 2)
-    );
-    inset-inline-end: calc(
-      calc(var(--spectrum-textfield-validation-icon-width) / 2) -
-        calc(var(--spectrum-icon-alert-medium-width) / 2)
+      calc(var(--spectrum-textfield-height) / 2) -
+        calc(var(--spectrum-textfield-validation-icon-height-error) / 2)
     );
 
-    padding-inline-end: calc(
-      var(--spectrum-textfield-padding-x) / 2 -
-        var(--spectrum-global-dimension-size-10)
-    );
+    inset-inline-end: var(--spectrum-textfield-icon-margin-left);
```

Comment:
Why change from padding to inset?

Reply:
So it used padding and positioning to get it offset but the valid icon doesn't. This gets the icon in the same place, but just using tokens and not using global dimensions.